packages/sanes-chrome-extension/src/logic/persona/persona.unit.spec.ts

```diff
@@ -58,6 +58,28 @@ withChainsDescribe('Persona', () => {
     });
   });
 
+  describe('createAccount', () => {
+    it('can create an account', async () => {
+      const persona = await Persona.create();
+
+      {
```

Comment:
It creates a new scope, such that scoped variables (`let`, `const`) are only available in this scope. E.g.

```js
const a = 1;

// a exists here

{
  const b = 2;
  // a exists here, b exists here
}

// a axists here, but no b anymore, since b went out of scope.
```

This is useful in tests when you want to reuse the same variable names multiple times. Here I have 2 `const accounts`.